examples/select-table-row-by-cell-content/README.md

```diff
@@ -0,0 +1,29 @@
+# Select a Table Row by Cell Content
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to select a table row based on the cell content.
+
+The tested page includes a table with `Id`, `First name` and `Last name` columns and three rows of data.
+A click on the row highlights it with a `highlighted` CSS class.
+
+The test searches the table by the value of its `Id` column to find the desired row and click it. Then it asserts that the row is highlighted, and has expected cell values.
```

Comment:
The test searches the table by the value of its `Id` column to find the desired row and click it. Then it asserts that the row is highlighted, and has expected cell values.
During the test the expected row data is declared as a JavaScript object. The `t.eql` method compares the object values with actual values found in each of the cells and check that the row has a `highlighted` class.
->
The test searches the target row by the value of its `Id` column. Then it asserts that the row is highlighted, and has expected cell values using the `t.eql` method. The expected cell values stored as a constant.

Reply:
The test identifies the target row by the value of its `Id` column. Then it uses the `t.eql` method to assert that the row has expected cell values and the `t.ok` method to check that it is highlighted.